apps/wallet/src/ui/app/components/menu/content/MenuListItem.tsx

```diff
@@ -51,6 +51,7 @@ function MenuListItem({
     if (href) {
         return (
             <a
+                data-testid={title}
```

Comment:
Why doesn't finding by text work given this title is still in the DOM?

Reply:
removed this